File: api/Pointage.js
"use strict";
class Pointage {
    constructor(manches, game, echange, final) {
        this.manches = manches;
        this.game = game;
        this.echange = echange;
        this.final = final;
    }
    static parse(pointage) {
        return new Pointage(pointage.manches, pointage.jeu, pointage.echange, pointage.final);
    }
}

File: original/android/app/src/main/java/ca/usherbrooke/bonpari/api/Pointage.kt
package ca.usherbrooke.bonpari.api

import com.squareup.moshi.Json
import java.io.Serializable

data class Pointage(
    @Json(name = "manches") val manches:List<Int>,
    @Json(name = "jeu") val game:List<List<Int>>,
    @Json(name = "echange") val echange:List<Int>,
    @Json(name = "final") val final: Boolean)  : Serializable
